### DISCUSSION TITLE
Change Username an Password

### DISCUSSION THREAD

**Discussion by RamsukKockoi** [thomluther/ha-anker-solix#80] [Q&A] [answered]
Hi,

I want to controll some of the functions of my Anker Solix. For that I need to add the Main user and get rid of the invited one. It seems, that this is only possible by deleting the old login. Unfotunately, this deletes all Data und and a massive Number of sensors, i built for the integration.

Is there a smarter way to change the user?

Thanks in advance
Ramsuk

**Comment by thomluther** [comment, marked as answer]
That is no issue, just delete the config entry and create it with the main account. Your entities will all re-appear with history and the new entities as well.
Their unique id will not change. I did that multiple times with my accounts

**Comment by RamsukKockoi** [comment]
Thank you very much. It worked.
